src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -144,18 +144,15 @@ const Prescriptions = () => {
     scrollLocation?.current?.scrollIntoView();
   };
 
-  useEffect(
-    () => {
-      if (!isLoading) {
-        if (prescriptionId) {
-          goToPrevious();
-        } else {
-          focusElement(document.querySelector('h1'));
-        }
+  useEffect(() => {
+    if (!isLoading) {
+      if (prescriptionId) {
```

Comment:
There doesn't seem to be anything in the !isLoading condition other than the other if branch, what do you think about consolidating the two? like this `if (!isLoading && prescriptionId) {}`

Reply:
I did not change this part. Just an indentation. 

Nevertheless, taking into account the presence of `else` statement inside, following your suggestion would influence the existing logic, so I would keep this bit as it is.

Reply:
Makes sense 👍